instrumentation/otel-webserver-module/include/core/sdkwrapper/IScopedSpan.h

```diff
@@ -31,7 +32,14 @@ using namespace opentelemetry;
 
 enum class StatusCode {
   Ok,
-  Error
+  Error,
+  Unset
+};
+
+enum class SpanType {
+  INTERNAL,
+  SERVER,
+  CLIENT
 };
```

Comment:
We already have SpanKind defined in ISdkWrapper. Any Challenge in re-using that?

Reply:
Since they are in ISdkWrapper.h, including that in this header file is leading to circular inclusions. So avoided it.

Reply:
ISdkWrapper is the parent right? Why would there be a problem of circular inclusion?

Reply:
Moved them to a new file to avoid circular inclusion